maat-court-data-api/src/main/java/gov/uk/courtdata/prosecutionconcluded/service/ProsecutionConcludedDataService.java

```diff
@@ -52,7 +68,7 @@ public void updateConclusion(Integer maatId) {
         List<ProsecutionConcludedEntity> processedCases = prosecutionConcludedRepository.getByMaatId(maatId);
         processedCases.forEach(concludedCase -> {
             concludedCase.setStatus(CaseConclusionStatus.PROCESSED.name());
-            concludedCase.setCreatedTime(LocalDateTime.now());
+            concludedCase.setUpdatedTime(LocalDateTime.now());
```

Comment:
@nittagr it looks like this was the source of the error seen in [LASB-1674](https://dsdmoj.atlassian.net/browse/LASB-1674) but @msahilan07 has kindly fixed it for us already! I check the database and there is no evidence of the created/updated time mismatch since this was merge in Sept 2022.

Reply:
@mtac50 - Good spot Mike :-) , I will add the note in the bug and close it.